practice-one/src/assets/styles/bases/typography.css

```diff
@@ -31,3 +31,19 @@ a {
   font-size: var(--font-size-xs);
   line-height: 24px;
 }
+
+/*-----------------------------------------*\
+  #SET FONT FAMILY
+\*-----------------------------------------*/
+
+@font-face {
+  font-family: "Abel";
+  font-style: normal;
+  src: url("../fonts/Abel-Regular.ttf") format("truetype");
+}
+
+@font-face {
```

Comment:
@HoangLiarPhuongLe ,
I think you should add `font-weight` property with a value of 400 to specify the normal font weight.
This helps ensure consistency and compatibility with different browsers and rendering engines.

Reply:
@HoangLiarPhuongLe Please check this comment!